src/mui/delete/Delete.js

```diff
@@ -42,21 +43,26 @@ class Delete extends Component {
     }
 
     render() {
-        const { title, id, data, isLoading, resource } = this.props;
+        const { title, id, data, isLoading, resource, translate } = this.props;
         const basePath = this.getBasePath();
+        const deleteItemLabel = translate(
+            'aor.action.delete_item',
+            { name: `${inflection.humanize(inflection.singularize(resource))} #${id}` },
```

Comment:
Note: It is not perfect since the resource name isn't translated, so we'll end up with view titles like:

> Delete Shoe #1 

> Supprimer Shoe #1 

The right way to to it is to replace the name *before* translation. e.g.:

```js
// pseudo-code
// in messages.js
messages['aor.action.delete_item'] = 'Delete %{name} #%{id}'

// first step: replace %name with inflection of resource
const title = messages['aor.action.delete_item'].replace('{%name}', inflection.humanize(inflection.singularize(resource)))

// second step: translate it
const deleteItemLabel = translate(title, { id: record.id });
```

You probably need a new helper in addition to `translate` to do so.